database/mysql/mysql.go

```diff
@@ -13,15 +13,11 @@ import (
 	nurl "net/url"
 	"strconv"
 	"strings"
-)
 
-import (
 	"github.com/go-sql-driver/mysql"
-	"github.com/hashicorp/go-multierror"
-)
-
-import (
 	"github.com/golang-migrate/migrate/v4/database"
+	"github.com/hashicorp/go-multierror"
```

Comment:
Related to my previous comment about the upstream repo. I think we should maintain their code style here, so I would suggest to revert this. But only if it looks like the other files are following a specific format.